pkg/authorization/api/v1/types.go

```diff
@@ -146,6 +146,27 @@ type SelfSubjectRulesReviewSpec struct {
 	Scopes OptionalScopes `json:"scopes" protobuf:"bytes,1,rep,name=scopes"`
 }
 
+// SubjectRulesReview is a resource you can create to determine which actions another user can perform in a namespace
+type SubjectRulesReview struct {
+	unversioned.TypeMeta `json:",inline"`
+
+	// Spec adds information about how to conduct the check
+	Spec SubjectRulesReviewSpec `json:"spec" protobuf:"bytes,1,opt,name=spec"`
+
+	// Status is completed by the server to tell which permissions you have
+	Status SubjectRulesReviewStatus `json:"status,omitempty" protobuf:"bytes,2,opt,name=status"`
+}
+
+// SubjectRulesReviewSpec adds information about how to conduct the check
+type SubjectRulesReviewSpec struct {
+	// User is optional.  At least one of User and Groups must be specified.
```

Comment:
> Why not an array of Subjects? Since this is SubjectRulesReview

I was matching `SubjectAccessReviewSpec`.  I would expect these look the same in `v1` and we'd change them all in `v2`.